datahub-web-react/src/app/search/SearchBar.tsx

```diff
@@ -278,13 +278,15 @@ export const SearchBar = ({
             >
                 <StyledSearchBar
                     placeholder={placeholderText}
-                    onPressEnter={(e) => {
-                        e.stopPropagation();
+                    onPressEnter={() => {
+                        // e.stopPropagation();
                         onSearch(filterSearchQuery(searchQuery || ''));
                     }}
                     style={inputStyle}
                     value={searchQuery}
-                    onChange={(e) => setSearchQuery(e.target.value)}
+                    onChange={(e) => {
+                        setSearchQuery(e.target.value);
```

Comment:
why is this change made?

Reply:
We were testing something on it. It's not changed to previous one.